gobcore/quality/issue.py

```diff
@@ -342,10 +344,14 @@ def _start_issue_workflow(header, issues, quality_update):
 
             wf_msg = {
                 'header': quality_update.get_header(header),
-                'contents_ref': writer.filename,
+                'contents_ref': filename,
                 'summary': {
                     'num_records': num_records
                 }
             }
 
             start_workflow(workflow, wf_msg)
+
+    if not num_records:
+        # no offloading, delete empty file
```

Comment:
Maybe wrap this in a try/except to prevent the program to crash when something fails.